docs/apv.md

```diff
@@ -88,7 +88,7 @@ verdere intentie tot roleplay is niet toegestaan. Dit geld ook voor dieren.
 
 8. Hij/zij die geen waarde hecht aan het leven van het karakter dat hij/zij speelt (No Value of Life), zal
 bestraft worden. Wegrijden met een voertuig waarbij je trager als 20 km/h rijdt valt hier ook onder.
-   * Bij overtreding van lid 7 en 8 zal dit resulteren in een straf volgens de 2e categorie.
+   * Bij overtreding van lid 8 en 9 zal dit resulteren in een straf volgens de 2e categorie.
```

Comment:
Klopt dit wel? Lid 9 is namelijk ERP, en weet niet of deze 2 leden er mee bedoelt worden?

Reply:
Lijkt errop dat dit niet klopt, want lid 9 gaat over ERP Waar  een 7e categorie op staat, hiermee zou je daar dus afhaneklijk van of je naar lid 8 of 9 kijkt een straf van 2e categorie of 7e categorie op staan.